database/seeds/UsersTableSeeder.php

```diff
@@ -18,12 +18,31 @@ public function run()
         $this->createCollegist();
         $this->createTenant();
         $this->createStaff();
-
+        
+        //generate random collegists
         factory(App\User::class, 10)->create()->each(function ($user) {
             factory(App\MacAddress::class, $user->id % 5)->create(['user_id' => $user->id]);
             factory(App\PrintJob::class, $user->id % 5)->create(['user_id' => $user->id]);
+            factory(App\PersonalInformation::class, $user->id % 5)->create(['user_id' => $user->id]);
+            factory(App\EducationalInformation::class, $user->id % 5)->create(['user_id' => $user->id]);
             $user->roles()->attach(Role::getId(Role::COLLEGIST));
             $user->roles()->attach(Role::getId(Role::INTERNET_USER));
+            for ($x = 0; $x < rand(1,3); $x++) {
+                $user->faculties()->attach(rand(1,7));
+            }
+            for ($x = 0; $x < rand(1,3); $x++) {
```

Comment:
You could use the same loop here.

Reply:
I used different loops to make it more random (this way, the user can have different number of faculties and workshops)